keps/sig-cluster-lifecycle/kubeadm/20191229-kubeadm-move-out-of-kk.md

```diff
@@ -0,0 +1,594 @@
+---
+title: Moving kubeadm out of kubernetes/kubernetes
+authors:
+  - "@neolit123"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-cluster-lifecycle
+reviewers:
+  - "@timothysc"
+  - "@fabriziopandini"
+  - "@rosti"
+  - "@ereslibre"
+  - "@detiber"
+  - "@yastij"
+  - "@LucaLanziani"
```

Comment:
@neolit123 -- Can you add @tpepper and myself as reviewers on this?

Reply:
@justaugustus i will add you as reviewers with the first batch of feedback.

Reply:
SGTM!